helm/openwhisk/values.yaml

```diff
@@ -259,6 +265,54 @@ controller:
   jvmOptions: ""
   loglevel: "INFO"
 
+# Scheduler configurations
+scheduler:
+  enable: true
+  imageName: "hunhoffe/scheduler"
```

Comment:
TODO: need to update this once apache/openwhisk has accepted changes needed for scheduler to replicate properly, and also once scheduler images are pushed to dockerhub.